Sources/MarathonCore/URLTransformer.swift

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Marathon
+ *  Copyright (c) Krzysztof Kapitan 2017
+ *  Licensed under the MIT license. See LICENSE file.
+ */
+
+import Foundation
+
+final class URLTransformer {
```

Comment:
How about adding this functionality to the already existing `URL+Marathon` extension? Might be easier to navigate the project if all URL-related functionality is in one place 😇